src/components/pages/ServiceReleaseProcess/components/ServiceDeactivate.tsx

```diff
@@ -0,0 +1,196 @@
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+import { PageBreadcrumb } from 'components/shared/frame/PageBreadcrumb/PageBreadcrumb'
+import {
+  Typography,
+  PageHeader,
+  Card,
+  Checkbox,
+  Button,
+  Tooltips,
+  LoadingButton,
+} from '@catena-x/portal-shared-components'
+import { useTranslation } from 'react-i18next'
+import { useLocation, useNavigate, useParams } from 'react-router-dom'
+import { Box } from '@mui/material'
+import { useCallback, useEffect, useState } from 'react'
+import { error } from 'services/NotifyService'
+import {
+  ServiceDeactivateEnum,
+  useDeactivateServiceMutation,
+  useFetchDocumentMutation,
+} from 'features/serviceManagement/apiSlice'
+import { DocumentTypeId } from 'features/appManagement/apiSlice'
+import { PAGES } from 'types/Constants'
+
+export default function ServiceDeactivate() {
+  const { t } = useTranslation('servicerelease')
+  const navigate = useNavigate()
+  const serviceId = useParams().serviceId
+  const [checked, setChecked] = useState(false)
+  const { state } = useLocation()
+  const [isLoading, setIsLoading] = useState(false)
+  const items: any = state
+  const service = items?.filter((item: any) => item.id === serviceId)
+  const [deactivateService] = useDeactivateServiceMutation()
+  const [fetchDocumentById] = useFetchDocumentMutation()
+  const [deactivateCardImage, setDeactivateCardImage] = useState('')
+  const leadImageId = service?.[0]?.leadPictureId
+
+  const fetchImage = useCallback(
+    async (documentId: string, docType: string) => {
+      try {
+        const response = await fetchDocumentById({
+          appId: serviceId ?? '',
+          documentId,
+        })
+
+        if (response && 'data' in response) {
+          const file = response?.data?.data
+          if (docType === DocumentTypeId.APP_LEADIMAGE) {
+            return setDeactivateCardImage(URL.createObjectURL(file))
+          }
+        }
+      } catch (err) {
+        error('ERROR WHILE FETCHING IMAGE', '', err as object)
+      }
+    },
+    [fetchDocumentById, serviceId]
+  )
+
+  useEffect(() => {
+    if (leadImageId) {
+      fetchImage(leadImageId, DocumentTypeId.APP_LEADIMAGE)
+    }
+  }, [fetchImage, leadImageId])
```

Comment:
Don't write custom fetch image code - use shared <Image> component and pass url along with ´ImageService.fetchImageWithToken´.

Reply:
@oyo Addressed comments